### ISSUE TITLE
Add focus 'modality' polyfill?

### ISSUE THREAD

**Issue by necolas** [necolas/react-native-web#310]
mobile.twitter.com uses a modified modality polyfill based on https://github.com/WICG/modality. It prevents focus rings from showing unless the keyboard is in use, which improves the mouse/touch UX. Consider whether it should be included in RNW by default or recommended as an add-on.
